specs/protocol/README.md

```diff
@@ -306,18 +306,10 @@ A `Balance` is a protected resource used to communicate the amounts of each curr
     "protocol": "1.0"
   },
   "data": {
-    "balances": [
-      {
-        /** ISO 4217 currency code or widely adopted cryptocurrency code as string */
-        "currencyCode": "USD", 
-        /** same format used to represent currency values across messages */
-        "available": "400.00",
-      },
-      {
-        "currencyCode": "MXN", 
-        "available": "100.00",
-      }
-    ]
+    /** ISO 4217 currency code or widely adopted cryptocurrency code as string */
```

Comment:
with this change, does it mean Alice can only ever have 1 currency balance stored with a PFI?

Reply:
the above spec change says `data: Balance[]` - so should this example show an array of balance details?

Reply:
This is defining what a single balance object looks like. So each balance object has:
```
{
  metadata: {}
  data: {
     "currencyCode": "USD",
     "available": "400.00"
  }
  signature: "abc123"
}
```

Alice can have multiple associated balances.

Reply:
can you write the example to be the balance message I'd get back if I had multiple balances with a PFI?